src/autosuggest/__integ__/events-autosuggest.test.ts

```diff
@@ -25,14 +25,15 @@ describe.each<boolean>([false, true])('Autosuggest events (expandToViewport=%s)'
   );
 
   test(
-    'should fire change event when selecting suggestions with SPACE',
+    'should allow entering spaces after focusing a dropdown item',
```

Comment:
The test logic did not change, but the title was misleading. Space key does not select any options (before and after this PR)